engine/test/integration_tests/dm_tls/run.sh

```diff
@@ -46,8 +66,13 @@ function run() {
 	sed -i "s,<mysql2-key>,$(base64 -w0 $WORK_DIR/mysql2/client-key.pem)," $WORK_DIR/job.yaml
 	sed -i "s,<mysql2-cert>,$(base64 -w0 $WORK_DIR/mysql2/client-cert.pem)," $WORK_DIR/job.yaml
 
+	# wait executor online
```

Comment:
It is the error that interrupting the executor to register itself, so we don't need to sleep such a long time.
May be `3s` is enough, or there exists a bug inside the engine.

Reply:
I checked that after https://github.com/pingcap/tiflow/commit/f4c5ed92c4cc6838493c36f51624b232d29074fc we the retry number is 6, which is 6 seconds.